specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/recoverableManagedDatabases.json

```diff
@@ -0,0 +1,283 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-10-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
```

Comment:
Is it possible to delete a recoverableDatabase through an API? Or is this a valid scenario? I am assuming that databases are automatically categorized as recoverable or not by the service.

Reply:
Recoverabledatabase is subtype of databases. If customer want to drop recoverable database, they can use database api to drop it. thanks

Reply:
@ravbhatnagar their lifetime is managed by the service, so no create or delete API is possible or needed.